src/studio/src/designer/backend/Services/Implementation/SourceControlSI.cs

```diff
@@ -645,5 +661,37 @@ public void AbortMerge(string org, string repository)
                 }
             }
         }
+
+        /// <inheritdoc/>
+        public async Task<RepositoryClient.Model.Branch> CreateBranch(string org, string repository, string branchName)
```

Comment:
I really like these three methods, small, concise and each with one clear responsibility.